### DISCUSSION TITLE
Minimap

### DISCUSSION THREAD

**Discussion by AndreM222** [akiyosi/goneovim#444] [Ideas]
Hey man is awesome and good job. I was wondering if you could make a separate plugin for the minimap by itself.

**Comment by akiyosi** [comment]
Is this an idea to separate minimap from the functionality of goenovim and make it a plugin?
Is there a motivation to separate it out, e.g., because there are users who don't need it in the main unit, and they are not satisfied with disabling it in the configuration file?

**Reply by AndreM222** [reply]
just an idea. Mostly because I like your minimap
